src/components/keymap/header/keymap-header.component.ts

```diff
@@ -14,10 +14,9 @@ import { KeymapActions } from '../../../store/actions';
 })
 export class KeymapHeaderComponent {
     @Input() keymap: Keymap;
+    @Output() deleteClick = new EventEmitter();
```

Comment:
If the keymap-edit don't need to set if a delete has already happened or not, then this can be removed, too.